client/src/Components/Header.js

```diff
@@ -108,9 +108,14 @@ function SearchAppBar(props) {
           </div>
           {
             props.isLogged ? (
-              <Button onClick={(event) => {props.logout(event)}} className={classes.button}>
-                Logout
-              </Button>
+              <div>
+                <span>
+                  What's up, {props.firstName}! 
```

Comment:
We should look into selecting random greetings. That would be dope. Other than that good job